mito-ai/ui-tests/tests/mitoAdvice.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { expect, test } from '@jupyterlab/galata';
+
+test('should display an advice message on empty cell', async ({ page }) => {
+  await page.notebook.createNew();
+  await page.notebook.enterCellEditingMode(0);
+
+  // Should display the advice message by default
+  await expect
+    .soft((await page.notebook.getCellLocator(0))!.getByRole('textbox'))
+    .toHaveText(
+      'Press Ctrl + E to ask Mito AI to do something. Start typing to dismiss.'
+    );
+
+  await page.notebook.setCell(0, 'code', 'print("Hello, World!")');
+
+  await expect
+    .soft((await page.notebook.getCellLocator(0))!.getByRole('textbox'))
```

Comment:
ooo cool! I didn't know this soft assertion was a thing. This is great for letting us test each of the different permutations in one test. I like it.

Reply:
Yep and it is recommended as best practice: https://playwright.dev/docs/best-practices#use-soft-assertions